src/components/LocationMap/LocationMap.js

```diff
@@ -17,24 +17,18 @@ const SimpleMap = withGoogleMap(props =>
 
 export default class LocationMap extends Component {
 
-	//this functionality should be moved to an epic:
-	// componentDidMount(nextProps) {
-	// 	navigator.geolocation.getCurrentPosition(position => {
-	// 		const latLng = {
-	// 			lat: position.coords.latitude,
-	// 			lng: position.coords.longitude
-	// 		}
-
-	// 		this.mapLib.panTo(latLng)
-	// 	})
-	// }
-
+		componentDidMount(nextProps) {
+      this.props.getUserCurrentLocation()
+      // this.mapLib.panTo(this.props.latLng)
```

Comment:
I'm ok with getting this merged in and doing the panning to the user's location in another PR. We could create another task for finishing off the map, which could include panning and also letting the user click on a marker and seeing the store details (I assume we are going to include that?)